### PR TITLE
always send json formatted logs to the telemetry service

### PR DESCRIPTION
**Pull request by aluon**
## Description

Change logger to always send json formatted logs to the telemetry service

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## How Has This Been Tested?

`cargo test`

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by aluon** [comment]
* **#12639** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-core/12639?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @aluon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by ibalajiarun — APPROVED** [review]

**Review by sionescu — APPROVED** [review]

File: crates/aptos-logger/src/aptos_logger.rs [hunk 1]
```diff
@@ -724,6 +724,28 @@ impl Writer for FileWriter {
     }
 }
 
+pub struct MockWriter {
```
Comment (brianolson) [inline crates/aptos-logger/src/aptos_logger.rs:727]: these could also be under `#[cfg(test)]` hiding?

Comment (brianolson) [reply crates/aptos-logger/src/aptos_logger.rs:727]: Or inside the tests mod?

Comment (aluon) [reply crates/aptos-logger/src/aptos_logger.rs]: Moved this into the tests module

**Review by brianolson — APPROVED** [review]

**Pull request merged**